benchmark/benchmark-tidb-using-tpcc.md

```diff
@@ -68,23 +68,47 @@ For example, the hardware configuration is as follows:
 | RAM | 128GB |
 | DISK | Optane 500GB SSD |
 
-1. Because this type of CPU has an NUMA architecture, it is recommended to bind the core using `taskset` and view the NUMA node using `lscpu`. For example:
+1. Because this type of CPU has an NUMA architecture, it is recommended to bind the core using `numactl`.
```

Comment:
```suggestion
1. Because this type of CPU has a NUMA architecture, it is recommended to [bind the core using `numactl`](/check-before-deployment.md#install-the-numactl-tool).
```